src/style.css

```
@@ -0,0 +1,106 @@
+:root {
+  --c-grey: #3d4e61;
+  --c-light: #e1e7ed;
+}
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+  font-style: normal;
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+  font-style: normal;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.input {
+  display: block;
+  width: 100%;
+  position: relative;
+  height: 70px;
+  padding-left: 62px;
+  border: 1px solid var(--c-light);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 16px;
+  padding-top: 23px;
+  padding-bottom: 25px;
+}
+
+.input::placeholder {
+  font-family: "Avenir", sans-serif;
+  color: var(--c-grey);
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  position: relative;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid var(--c-light);
+  border-radius: 3px;
+  color: var(--c-grey);
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+}
+
+.input__icon {
+  position: absolute;
+  z-index: 1;
+  top: 42px;
+  left: 26px;
+}
+
+.input.input--small {
+  position: relative;
+  height: 42px;
+  padding-top: 10px;
+  padding-bottom: 12px;
+  padding-left: 33px;
+  box-sizing: border-box;
+  color: var(--c-grey);
+  border: 1px solid var(--c-light);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-size: 14px;
+}
```

point 2 - https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md

Also, increase specificity here is needless
```suggestion
.input--small {
  position: relative;
  height: 42px;
  padding-top: 10px;
  padding-bottom: 12px;
  padding-left: 33px;
  box-sizing: border-box;
  color: var(--c-grey);
  border: 1px solid var(--c-light);
  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
  border-radius: 4px;
  font-size: 14px;
}
```